test_util/cluster.py

```diff
@@ -0,0 +1,260 @@
+import itertools
+import logging
+import os
+import random
+import stat
+import string
+from collections import namedtuple
+from contextlib import closing, contextmanager
+from os.path import join
+
+import passlib.hash
+import pkg_resources
+from retrying import retry
+
+import test_util.ccm
+import test_util.installer_api_test
+import test_util.test_runner
+from ssh.ssh_tunnel import SSHTunnel, TunnelCollection
+
+
+Host = namedtuple('Host', ['public_ip', 'private_ip'])
```

Comment:
I'll be sure to include it when I update https://github.com/dcos/dcos/pull/622 after this has merged